assets/core/css/custom.css

```diff
@@ -3,4 +3,5 @@
    Custom Stylesheet
 
    All changes made in this file will override the default styles.
-   ========================================================================== */
\ No newline at end of file
+   ========================================================================== */
+   .model-pager-detailed{text-align: center;}
```

Comment:
Whoops, no CSS into the custom.css please.

Reply:
But if I add in the css of the core, I have to eliminate the gitignore to be able to synchronize it. Or how is it done?

Reply:
Just remove it, I will add it to the corresponding file then.

Reply:
ok, what would be the correct way to do it ?, what file ?, for the next time.